packages/date-picker/src/vaadin-date-picker-styles.js

```diff
@@ -10,11 +10,6 @@ export const datePickerStyles = css`
     direction: ltr;
   }
 
-  :host([dir='rtl']) [part='value']::placeholder {
-    direction: rtl;
-    text-align: left;
-  }
```

Comment:
These styles are no longer used since #2532 where we changed to use slotted `<input>` element.